lib/MiLightState/GroupState.cpp

```diff
@@ -297,6 +298,17 @@ bool GroupState::setNightMode(bool nightMode) {
   return true;
 }
 
+bool GroupState::isPendingSaturation() const { return state.fields._isPendingSaturation; }
```

Comment:
Now that it's optional, I think it's fine for it to behave the way you had it in the original PR.

The concerns I have with attaching it to state are:

* We're having to add another byte, which might cause issues with backwards compatibility
* This isn't really a persistent state field, so it feels a little weird putting it here
* Because the field is ignored when loaded, things might behave strangely if a bulb's state is flushed to flash (a limited number of states are kept in RAM)

It feels like if we want to go down this road, we should do it for every field, and I think the approach we'd take would look a little different if we did.